src/data/migrations/20190822155403_create_steps_table.js

```diff
@@ -0,0 +1,25 @@
+exports.up = function(knex) {
+    return knex.schema.createTable("steps", steps => {
+        steps.increments()
+        steps
+            .integer("number")
+            .unsigned()
+            .notNullable()
+        steps.text("description").notNullable()
+        steps.boolean("is_required").notNullable()
+        steps
+            .integer("tasks_id")
+            .unsigned()
+            .notNullable()
+            .references("id")
+            .inTable("tasks")
+            .onDelete("CASCADE")
+            .onUpdate("CASCADE")
+
+        steps.unique(["tasks_id", "number"])
```

Comment:
Each task should only have one step 1, one step 2 etc, so I made this relationship between tasks_id and number unique, we didn't discuss this as a group let me know if there is something wrong with it